src/masternodes/govvariables/attributes.cpp

```diff
@@ -274,6 +327,24 @@ Res ATTRIBUTES::Validate(const CCustomCSView & view) const
                     if (!view.GetLoanTokenByID(DCT_ID{tokenId})) {
                         return Res::Err("No such loan token (%d)", tokenId);
                     }
+                } else if (attrV0->key == TokenKeys::LoanCollateralEnabled ||
+                           attrV0->key == TokenKeys::LoanCollateralFactor ||
+                           attrV0->key == TokenKeys::LoanMintingEnabled ||
+                           attrV0->key == TokenKeys::LoanMintingInterest) {
+                    if (view.GetLastHeight() < Params().GetConsensus().GreatWorldHeight) {
+                        return Res::Err("Cannot be set before GreatWorld");
+                    }
+                } else if (attrV0->key == TokenKeys::FixedIntervalPriceId) {
+                    if (view.GetLastHeight() < Params().GetConsensus().GreatWorldHeight) {
+                        return Res::Err("Cannot be set before GreatWorld");
+                    }
+                    if (const auto currencyPair = std::get_if<CTokenCurrencyPair>(&attribute.second)) {
+                        if (!OraclePriceFeed(const_cast<CCustomCSView&>(view), *currencyPair)) {
```

Comment:
Could be moved to apply ?

Reply:
Does not apply anything here, only validates.

Reply:
Yup, just only to skip `const_cast`

Reply:
Got it, good thinking.